config/initializers/stake_pools_data.rb

```diff
@@ -92,10 +92,10 @@
     ticker: "jitosol",
     small_logo: "jito-logo.svg",
     large_logo: "jito.png",
-    url: "https://www.jito.network/",
-    manager_fee: 4, #https://jito-foundation.gitbook.io/jitosol/faqs/general-faqs#fees
+    url: "https://www.jito.network",
+    manager_fee: 4, #https://exponential.fi/protocols/72aec1c8-dd85-427d-bb50-5461347ec247
```

Comment:
```suggestion
    manager_fee: 4, #https://www.jito.network/docs/jitosol/general-faqs
```

We do not want to rely on third parties for our data